src/display.cpp

```diff
@@ -39,7 +39,12 @@ static const itype_id fuel_type_muscle( "muscle" );
 // Cache for the overmap widget string
 static disp_overmap_cache disp_om_cache;
 // Cache for the bodygraph widget string
-static disp_bodygraph_cache disp_bg_cache;
+static disp_bodygraph_cache disp_bg_cache[] = {
```

Comment:
Is there something like list comprehensions to directly construct this from the enum in a more elegant way? (not a C++ expert...)